docs/orphaned_and_conflicting.md

```diff
@@ -0,0 +1,82 @@
+In this guide, we will look at how to deal with the files in `files/<locale>/orphaned` and `files/<locale>/conflicting`. Ideally, those folders should be empty, but in practice, some regular maintenance is needed from each localization team and some more work might be needed if this has not been done since the MDN migration to GitHub.
+
+## What they are
+
+Yari relies on file structure/hierarchy being the same between `files/en-us/` from `mdn/content` and `files/<locale>/` from `mdn/translated-content`. However, from time to time, some differences occur and must be solved by a humain maintainer. Such differences are pushed into `files/<locale>/orphaned` and `files/<locale>/conflicting` directories.
+
+### Orphaned pages
+
+When a page does not exists anymore in English (or didn't exist at all, like it was possible to have in Kuma) but exists in a given locale, it is moved to `files/<locale>/orphaned`. The locale team should then decide if this content should be deleted/redirected or migrated to another page.
+
+In practice, deleting-redirecting per the `en-US` structure is enough (see below).
+
+### Conflicting pages
+
+When an English page is redirected/moved to a new location, but the target already exists in `<locale>`, the localized page is moved to `files/<locale>/conflicting`. Basically, a locale team member should decide which version take priority over the other.
+
+## How they are generated
+
+This "structure" synchronization between `mdn/content` and `mdn/translated-content` occurs daily with this GitHub Actions workflow https://github.com/nschonni/mdn-translated-content/blob/main/.github/workflows/sync-translated-content.yml (the logic can be found at https://github.com/mdn/yari/blob/main/build/sync-translated-content.ts)
```

Comment:
Shouldn’t be MDN the org?

```suggestion
This "structure" synchronization between `mdn/content` and `mdn/translated-content` occurs daily with this GitHub Actions workflow https://github.com/mdn/mdn-translated-content/blob/main/.github/workflows/sync-translated-content.yml (the logic can be found at https://github.com/mdn/yari/blob/main/build/sync-translated-content.ts)
```

Reply:
Thanks, I guess I should not rely on opened tabs ^^